sqlserver/assets/configuration/spec.yaml

```diff
@@ -140,9 +140,9 @@ files:
       value:
         type: boolean
         example: false
-    - name: include_fci_metrics
```

Comment:
We can't change the configuration out from under customers. This should remain as-is.